### DISCUSSION TITLE
ServerlessBundler plugin should maybe have an option to prevent netlify.toml redirects to be of type 'force: true'

### DISCUSSION THREAD

**Discussion by PieturP** [11ty/eleventy#2136] [Enhancement Queue]
I think it would make sense to prevent the ServerlessBundler to inject `force: true` to redirects configs in the netlify.toml file. 

That way, if a file / route is already present, it would serve that, instead of invoking the serverless function or on-demand builder.

Consider an example website where there's thousands of records (say Books), but only a handful are mentioned on this site's homepage. The ones mentioned could simply be statically generated and served quickly. The items (Books) that are not mentioned could then be dynamically rendered upon some kind of search action of sorts.
I'm aware that Netlify on-demand builders fills this gap for a bit, but still...

I'm also not sure why this `force: true` is actually currently necessary. But I'm possibly missing something here.

Happy to hear any feedback! :-)

**Comment by zachleat** [comment]
Sorry this one didn’t get an answer! https://www.11ty.dev/docs/plugins/serverless/
